test/v2/models/request/common/ukFhlPieProperty/UkFhlPieIncomeSpec.scala

```diff
@@ -36,8 +34,7 @@ class UkFhlPieIncomeSpec extends UnitSpec{
       |}
       |""".stripMargin)
 
-  val desJson: JsValue = Json.parse(
-    """
+  val desJson: JsValue = Json.parse("""
```

Comment:
As this endpoint is connecting to IF, but one of the others is connecting to DES, could this common code be renamed to `downstreamJson`?  (Also worth using "downstream" instead of DES or IF, as a general rule)